### DISCUSSION TITLE
8.0版本无法授权使用专业版

### DISCUSSION THREAD

**Discussion by 904787103** [getlantern/lantern-forum-cn#619] [Q&A]
邮箱连接和pin码连接都不能用

**Comment by MaxMKMax** [comment]
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
@904787103 您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

我们将非常感谢您。
